cumulus/polkadot-parachain/src/chain_spec/asset_hubs.rs

```diff
@@ -14,59 +14,39 @@
 
-use crate::chain_spec::{get_account_id_from_seed, get_collator_keys_from_seed, SAFE_XCM_VERSION};
-use cumulus_primitives_core::ParaId;
+use crate::chain_spec::{Extensions, GenericChainSpec};
+use asset_hub_rococo_runtime::genesis_config_presets::{
+	asset_hub_rococo_development_genesis, asset_hub_rococo_genesis,
+	asset_hub_rococo_local_testnet_genesis,
+};
+use asset_hub_westend_runtime::genesis_config_presets::{
+	asset_hub_westend_development_genesis, asset_hub_westend_genesis,
+	asset_hub_westend_local_testnet_genesis,
+};
 use hex_literal::hex;
-use parachains_common::{AccountId, AuraId, Balance as AssetHubBalance};
+use parachains_common::Balance as AssetHubBalance;
 use polkadot_parachain_lib::chain_spec::{Extensions, GenericChainSpec};
 use sc_service::ChainType;
-use sp_core::{crypto::UncheckedInto, sr25519};
+use sp_core::crypto::UncheckedInto;
 
 const ASSET_HUB_WESTEND_ED: AssetHubBalance = asset_hub_westend_runtime::ExistentialDeposit::get();
 const ASSET_HUB_ROCOCO_ED: AssetHubBalance = asset_hub_rococo_runtime::ExistentialDeposit::get();
 
-/// Generate the session keys from individual elements.
-///
-/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn asset_hub_rococo_session_keys(keys: AuraId) -> asset_hub_rococo_runtime::SessionKeys {
-	asset_hub_rococo_runtime::SessionKeys { aura: keys }
-}
-
-/// Generate the session keys from individual elements.
-///
-/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn asset_hub_westend_session_keys(keys: AuraId) -> asset_hub_westend_runtime::SessionKeys {
-	asset_hub_westend_runtime::SessionKeys { aura: keys }
-}
-
 pub fn asset_hub_westend_development_config() -> GenericChainSpec {
 	let mut properties = sc_chain_spec::Properties::new();
 	properties.insert("tokenSymbol".into(), "WND".into());
 	properties.insert("tokenDecimals".into(), 12.into());
+	let para_id = 1000;
 
 	GenericChainSpec::builder(
 		asset_hub_westend_runtime::WASM_BINARY
 			.expect("WASM binary was not built, please build it!"),
-		Extensions { relay_chain: "westend".into(), para_id: 1000 },
+		Extensions { relay_chain: "westend".into(), para_id },
 	)
 	.with_name("Westend Asset Hub Development")
 	.with_id("asset-hub-westend-dev")
 	.with_chain_type(ChainType::Local)
-	.with_genesis_config_patch(asset_hub_westend_genesis(
-		// initial collators.
-		vec![(
-			get_account_id_from_seed::<sr25519::Public>("Alice"),
-			get_collator_keys_from_seed::<AuraId>("Alice"),
-		)],
-		vec![
-			get_account_id_from_seed::<sr25519::Public>("Alice"),
-			get_account_id_from_seed::<sr25519::Public>("Bob"),
-			get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-		],
-		testnet_parachains_constants::westend::currency::UNITS * 1_000_000,
-		1000.into(),
-	))
+	.with_genesis_config_patch(asset_hub_westend_development_genesis(para_id.into()))
```

Comment:
You could use: [`with_genesis_config_preset_name`](https://github.com/paritytech/polkadot-sdk/blob/6db203822418eee666f44ccaed3cd96fdfa27351/substrate/client/chain-spec/src/chain_spec.rs#L404).

This would allow to fetch the preset from the wasm blob, w/o necessity to natively compile the runtime.

I noticed the same in https://github.com/polkadot-fellows/runtimes/pull/379#pullrequestreview-2223232069

Reply:
yes, I saw that fellows comment, it is still in my TODOs :)